### ISSUE TITLE
Divider is invisible after clicking on image in Recycler View

### ISSUE THREAD

**Issue by mamatagelanee07** [HareshChhelana/CustomGalleryPickerForAndroid#2]
When we select and unselect images in custom gallary, space between two images is invisible. 

![issue](https://user-images.githubusercontent.com/8694335/34550792-a019cae4-f139-11e7-80d1-4a116df1ac54.gif)

**Comment by HareshChhelana** [comment]
Can you please share your item layout?

**Comment by tjohnn** [comment]
I just noticed the same issue here also which means the problem is from the source, I guess the issue has to do with the recyclerview that lists the gallery items.

**Comment by mamatagelanee07** [comment]
@HareshChhelana  Yes! @tjohnn is right. This code also has same issue so you can check your source!(P.S. change background of your `list_item_image.xml`) If you still need, let me know :)
